sources/web/datalab/polymer/components/directory-picker-dialog/directory-picker-dialog.ts

```diff
@@ -20,6 +20,7 @@
  */
 interface DirectoryPickerDialogCloseResult extends BaseDialogCloseResult {
```

Comment:
Now that you have the ability to pick a file as well, this class name is not quite accurate. Maybe not a big deal for now, but could be nice to change if you come up with a better and still succinct name.